.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+
+language: scala
+
+services:
+- docker
+
+cache:
+  directories:
+  - $HOME/.ivy2/cache
+  - $HOME/.sbt/boot/
+
+script:
+- mkdir -p ${HOME}/.sbt
+- docker run -v ${HOME}/.ivy2:/root/.ivy2 -v ${HOME}/.sbt:/root/.sbt -v ${PWD}:/usace-flood-geoprocessing
+  -w /usace-flood-geoprocessing quay.io/azavea/scala:2.10.5 ./sbt "project server" compile
+
+before_deploy:
+- mkdir -p ${HOME}/.sbt
```

Comment:
Should be able to omit this because `script` should run and succeed before `before_deploy` can be run.